lib/crawly/manager.ex

```diff
@@ -225,4 +215,13 @@ defmodule Crawly.Manager do
     do: String.to_integer(value)
 
   defp maybe_convert_to_integer(value) when is_integer(value), do: value
+
+  defp store_requests(spider_name, requests) do
```

Comment:
maybe rename to `store_requests_sync`

Reply:
I am not sure about it. Basically, I would only mark async functions, and assume everything else is sync.